litmus-portal/frontend/src/pages/AnalyticsPage/styles.ts

```diff
@@ -37,6 +38,17 @@ const useStyles = makeStyles((theme) => ({
   analyticsDiv: {
     marginTop: theme.spacing(5),
   },
+
+  waitingText: {
+    fontSize: '2rem',
+    marginLeft: '25%',
+    marginTop: '25%',
+    marginBottom: '10%',
+  },
```

Comment:
can you use themes for margin?

Reply:
I tried using theme but it's not responsive enough @Saranya-jena I will have to use breakpoints in that case and just for an error/loader message it will make styles file larger. can we keep it like this?